src/data/tags.ts

```diff
@@ -1,6 +1,13 @@
 import { Tag } from "../interfaces/Tag";
 
 export const tags: Tag[] = [
+  {
+    name: "contribute",
+    content:
+      "You can contribute to any repo with the hacktoberfest label, or if your PR gets the hacktoberfest-accepted label. You can find these by searching on either [GitHub](https://github.com/topics/hacktoberfest) or [GitLab](https://gitlab.com/explore/projects/topics/hacktoberfest).\n\nYou can contribute both code and low-code. See also [GitHub's guide](https://opensource.guide/how-to-contribute/) for contributing if you're not sure how. The [Hacktoberfest website](https://hacktoberfest.com/participation/#contributors) also contains some resources to help you get started.",
```

Comment:
```suggestion
      "You can contribute to any repo with the hacktoberfest label, or repos using the `hacktoberfest-label` on individual PRs/MRs. You can find these by searching on either [GitHub](https://github.com/topics/hacktoberfest) or [GitLab](https://gitlab.com/explore/projects/topics/hacktoberfest).\n\nYou can contribute both code and low-code. See also [GitHub's guide](https://opensource.guide/how-to-contribute/) for contributing if you're not sure how. The [Hacktoberfest website](https://hacktoberfest.com/participation/#contributors) also contains some resources to help you get started.",
```
Tweak the wording here a bit.

Reply:
I assume you typo'd and meant "[...] or repos using the `hacktoberfest-accepted` label on individual [...]"?